roles/confluent.ssl/tasks/custom_certs.yml

```diff
@@ -1,18 +1,33 @@
 ---
+- set_fact:
```

Comment:
i wonder if theres a way to get around these two set_facts ie in either confluent.ssl/defaults/main.yml or confluent.variables_and_handlers/defaults/main.yml just set a default and then that it, and for the user who sets ssl_custom_certs_skip_copy to true they would also have to update these filepath vars. LMK if that makes sense

Reply:
We’ll need a filter as far as I can see. Or get rid of the basename filter?